src/tiles/image-block.ts

```diff
@@ -94,6 +97,11 @@ export class ImageBlock extends LitElement {
         flex: 1;
       }
 
+      .grid.has-snippets {
+        /* If there is a text snippet block present, the image block needs to shrink */
+        height: 11rem;
```

Comment:
Suggestion: remove image block's awareness of "snippets" to keep component scope down.

we can probably control this height from the tile level? perhaps... toggle a CSS if tile has snippets by setting something like `style="var(--imgHeight: 11rem;)" whenever it needs it.

so in `.grid { }` rules, we can set default and entry like:
```
.grid { height: var(--imgBlockHeight,  16rem) };

Reply:
That makes more sense yeah -- I'll give this a try.